tests/PHPUnit/Integration/SegmentTest.php

```diff
@@ -1413,95 +1439,553 @@ public function test_getSelectQuery_whenPageUrlDoesNotExist_asStatementOR()
 
         $expected = array(
             "sql"  => "
-                SELECT
-                    log_visit.*
-                FROM
-                    " . Common::prefixTable('log_visit') . " AS log_visit
-                WHERE (HOUR(log_visit.visit_last_action_time) = ?
-                      OR (1 = 0) )",
-            "bind" => array(12));
+                SELECT log_inner.* FROM ( 
```

Comment:
> Also ensuring that when we use a == that it speeds up the query correctly where it effectively does a where idaction = $id.

> Curently not doing that, but instead, LEFT JOIN ON log_action idaction_url_log_action WHERE idaction_url_log_action.name = ? AND idaction_url_log_action.hash = CRC32(?) AND idaction_url_log_action.type = ?. This should have the same effect but require less queries, correct?

@diosmosis I believe here we can see that this used to be more efficient as in these cases we didn't need to join various tables.